index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="My landing page made for Coderscamp project">
+  <meta name="keywords" content="web design, html, css">
+  <meta name="author" content="Konrad Kalita">
+  <title>Landing Page</title>
+  <link rel="stylesheet" type="text/css" href="./css/styles.css">
+  <link rel="stylesheet" type="text/css" href="./css/buttons.css">
+  <link rel="stylesheet" type="text/css" href="./css/responsiveness.css">
+  <style>
+    @import url('https://fonts.googleapis.com/css?family=Raleway:400,700&display=swap');
+  </style>
+  <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  </head>
+  <body>
+    <header>
+      <nav class="container">
+        <div class="brand">
+          <img src="./img/logo_svg.png" alt="logo" height="26px" width="26px">
+          <h1 style="padding-left: 5px;">Dev</h1>
+        </div>
+        <ul class="navbar_buttons">
+          <li><a href="#about_me" class="navbutton">About me</a></li>
+          <li><a href="#skills" class="navbutton">Skills</a></li>
+          <li><a href="#portfolio" class="navbutton">Portfolio</a></li>
+          <li><a href="#contact_us" class="navbutton">Contact</a></li>
+        </ul>      
+      </nav>
+      <div class="container">
+        <div class="cover_photo">
+          <img src="./img/cover_layer.png" class="layer" alt="layer">
+          <h1>Welcome!</h1>
+          <h2>My name is Konrad.</h2> 
+          <div class="social_media">
+            <ul class="social_media">
```

Comment:
Dlaczego `div` oraz `ul` mają tą samą klase? Szczerze ? To patrząc po devtoolsach to te dwie klasy nie są nam nawet potrzebne :) (style nadane tej klasie są zbędne) a sama klasa nam jest potrzebna tylko dlatego że mamy nadane style tagu `li` -> `.social_media li`. Ale jeśli np `<li>` miałoby swoją klase to można byłoby usunąć `social_media`:

Ale skoro już np byśmy potrzebowali dla każdego z tagu klasę to proponowałbym taką strukturę: 
```
<div class="social_media__container">
  <ul class="social_media__list">
    <li class=""social_media__item></li>
  </ul>
</div>
```
😃